### PR TITLE
QUE: add question on dropout

### PR DESCRIPTION
**Pull request by Ishan-Kumar2**
A easy-level question on dropout and bagging.

### CODE REVIEW

File: questions/dl.yml [hunk 1]
```diff
@@ -20,4 +20,27 @@ questions:
...
   diff: easy
-  ref: NULL
\ No newline at end of file
+  ref: NULL
+
+  title: Discussing dropout 
+  type: mcq
+  text: Which of the following are true regarding dropout?
+  opt:
+    - Dropout is a method of bagging training.
```
Comment (AnirudhDagar) [inline questions/dl.yml:29]: Can we modify this?:
```suggestion
    - Dropout is similar to bagging training.
```
This is more clear I guess.

File: questions/dl.yml [hunk 2]
```diff
@@ -20,4 +20,27 @@ questions:
...
+  sol:
+    Dropout needs to be turned off during testing as we want the entire representational capacity of the model during testing. 
+    Dropout is similar to a bagging model but there are a few key differences. 
+    First in bagging the models are trained independently of each other whereas the dropout models share parameters with each other. 
+    Second, in bagging each individual model is trained to convergence while that is not the case in dropout (where each model is trained for 1 step).
+    Further D is incorrect since dropout helps reduce overfitting.
+  sol_img: NULL
+  tags: 
+    - regularization
+  diff: medium
```
Comment (AnirudhDagar) [inline questions/dl.yml:45]: easy? Maybe others can suggest.

**Review by AnirudhDagar — COMMENTED** [review]
@Ishan-Kumar2 made some comments even though this is directly from the book, i think we can make small changes to clarify the options.

**Review by AnirudhDagar — APPROVED** [review]
Thanks, Ishan! Sorry for the late response.

**Comment by AnirudhDagar** [comment]
@Garg-Doppler @saswatpp can you guys take a look and get this in if it looks good?

**Review by Garg-Doppler — APPROVED** [review]

**Pull request merged**